owner/manager.go

```diff
@@ -65,6 +69,25 @@ const (
 	keyOpDefaultTimeout = 5 * time.Second
 )
 
+// OpType is the owner key value operation type.
+type OpType byte
+
+// List operation of types.
+const (
+	OpNone              OpType = 0
```

Comment:
We can use ```iota``` here.

Reply:
I think it's a little bit clearer to just assign it